src/core/id-headers.js

```diff
@@ -20,8 +28,15 @@ export function run(conf) {
       id = h.parentElement.id || h.id;
     }
     if (!conf.addSectionLinks) continue;
-    h.appendChild(html`
-      <a href="${`#${id}`}" class="self-link" aria-label="§"></a>
+    let label = h.closest(".appendix")
+      ? l10n.perma_for_appendix
+      : l10n.perma_for_section;
```

Comment:
I feel it might be better if we have something like:
`label = l10n.permalinkLabel(type = "appendix" | "section", sectionNumber)`